example/ios/LiveMarkdownExample.xcodeproj/project.pbxproj

```diff
@@ -275,6 +275,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.LiveMarkdownExample.$(PRODUCT_NAME:rfc1034identifier)";
 				PRODUCT_NAME = LiveMarkdownExample;
+				SWIFT_OBJC_BRIDGING_HEADER = "LiveMarkdownExample-Bridging-Header.h";
```

Comment:
Do we need all these changes in `project.pbxproj` or can we just skip them?

Reply:
We can skip them [cbaebd4](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/cbaebd4853ef799b4b0f4de94d9568b0cd19f24a)